docs/operations-guide/creating-RDS-database-on-AWS.md

```diff
@@ -0,0 +1,98 @@
+  - [Step 1](#step-1-1)
+  - [Step 2](#step-2-1)
+  - [Step 3](#step-3-1)
+
+# Configuring RDS for Metabase (the recommended guide)
+
+If you want to move from using Metabase just for testing to something that is ready for the big time, you need to use a production-grade database like PostgreSQL or MySQL/MariaDB. This guide will provide you with all the steps for this venture. A high level architecture diagram of what you will achieve once you deploy Metabase with a dedicated application database can be [seen here](images/Metabase-AWS-SI.png)
+
+## Step 1
+In AWS, enter RDS in the search box or select the service from the dropdown button on the top left of the page. Once inside RDS, click on the `Create database` button
+
+## Step 2
+- Create Database: select MySQL or PostgreSQL as engine types, as these two are the ones that Metabase support as the Application Database (where Metabase will save all of its configurations). For this example we will choose PostgreSQL on its latest version available in AWS at this time (12.4-R1)
+
+- Templates: you can leave `Production` selected or choose any other option that better suits your needs.
+
+- Settings: type a unique `DB instance identifier` for your database. Also remember the username and a master password that you enter here as you will need those to configure the environment variables in Metabase.
+![RDS Templates Section](images/RDSPostgresSettings.png)
+
+- Instance size: you can choose any instance size that will fit your needs. As this is the application database for Metabase, the sizing of the RDS instance depends on the number of Metabase instances that will be connected to this database, number of users that are using Metabase at once and the number of questions, dashboards and configurations that are saved. To start, a `t3.small` is a good choice.
+![RDS Instance size](images/RDSInstanceSize.png)
+
+- Availability & Durability: on production deployments, you __should__ be using a Multi-AZ (Availability Zone) cluster, as this will ensure that the database does not goes down in case there is an issue on a single availability zone.
+![RDS MultiAZ](images/RDSMultiAZ.png)
+
+- Connectivity: 
+  - ensure that you are deploying the database in the same VPC as the one you deployed the Metabase instance/s, otherwise they won't be able to see each other. 
+  - create a __VPC security group__, as you will need to grant access from the Metabase instance/s to the database on the port that listens for connections.
+![RDS VPC Security Groups](images/RDSVPCSecurityGroup.png)
+
+- Additional configuration
+  - enter `metabase` as the `Initial database name` as this will be the database that Metabase will use for all of its configurations. 
+  - you can also configure the backup window in case you need to restore the backups at some point in time.
+![RDS Initial Database](images/RDSInitialDatabase.png)
```

Comment:
```suggestion
  - Enter `metabase` as the **Initial database name**. Metabase will use this database for all of its configurations. 
  - You can also configure the backup window in case you need to restore the backups at some point in time.
![RDS Initial Database](images/RDSInitialDatabase.png)
```